frontend/src/pages/ActivityPage.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import Card from '@material-ui/core/Card';
+import { Avatar, CardContent, CardHeader, Container, Paper, Typography } from '@material-ui/core';
+
+const fakeActivityData = {
+  title: "Muay Thai Session",
+  location: "My Garage",
+  desc: "A fun light paced one hour intro to muay thai.",
+  author: {
+    username: "Jay",
+    fullname: "Jay Gohner",
+    userId: 420,
+    avatarURL: "https://bullmuaythaikrabi.com/wp-content/uploads/2017/09/Bull-Muay-Thai-Krabi-fighters-news_6.jpg"
+  },
+};
```

Comment:
Following our style guide, JavaScript objects should use single quotes for strings